File: Programming-languages-and-laboratory/Python/Exercises/01_compito_java_20150119/Berlina.py
from Automobile import Automobile


class Berlina(Automobile):
    def __init__(self, tipo, modello, produttore, bagagliaio, peso, codice_identificativo):
        super().__init__(tipo, modello, produttore, peso, codice_identificativo)
        self._bagagliaio = bagagliaio

    def __str__(self):
        return super().__str__() + " " + str(self._bagagliaio) + " -"

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/01_compito_java_2015_01_19/Berlina.java
public class Berlina extends Auto{
    double bagagliaio;
    public Berlina(String m, String pr, double bag, int p, int cod){
        super(m, pr, p, cod);
        bagagliaio = bag;
    }

    public String toString(){
        return codice + "\t" + produttore + "\t" + modello + "\t" + bagagliaio + "\t-";
    }
}